libraries/AC_AttitudeControl/AC_AttitudeControl.h

```diff
@@ -376,6 +376,9 @@ class AC_AttitudeControl {
     // Sets the yaw rate shaping time constant
     void set_yaw_rate_tc(float input_tc) { _rate_y_tc = input_tc; }
 
+    // setup a one loop angle P scale
+    void set_angle_P_scale(const Vector3f &angle_P_scale) { _angle_P_scale = angle_P_scale; }
```

Comment:
I am considering setting this as a product rather than an equal. This would let multiple users access this for different purposes. For example:
Ground resonance suppression
Throttle slew boost

Most of the time everything is just not changing this and would just set it to 1. But this would mean the last user would reset it for the earlier users.

Reply:
added both APIs